test/Makefile

```diff
@@ -39,6 +45,12 @@
 #                        An empty set means only one permutation with no arguments.
 #                        default: the make variable ARGS (see below)
 #
+#   ARG_SETS:            sets off extra arguments to invoke $(DMD) with (seperated by ';').
+#                        default: (none)
```

Comment:
Ouch three more variables :/

- I don't see them separated by a `;` anywhere on your included/modified tests
- What's the difference to `REQUIRED_ARGS`? Couldn't ``REQUIRED_ARGS` be extended to accept multiple lines too?

Reply:
They can be seperated either by line or by `;`

REQUIRED_ARGS are used in every variation of the compilation, whereas an ARG_SET is a single set of extra arguments for one compilation.  It is a weaker form of PERMUTE_ARGS.